packages/client/src/pages/app/app.tsx

```diff
@@ -1,5 +1,116 @@
 import { useEffect } from 'react'
-import style from './app.module.scss'
+import {
+  createBrowserRouter,
+  isRouteErrorResponse,
+  Outlet,
+  redirect,
+  RouterProvider,
+  useRouteError,
+} from 'react-router-dom'
+import ErrorPage from '../error-page/error-page'
+import Forum from '../forum/forum'
+import Game from '../game/game'
+import Home from '../home/home'
+import Leaderboard from '../leaderboard/leaderboard'
+import Login from '../login/login'
+import Profile from '../profile/profile'
+import SignUp from '../signup/signup'
```

Comment:
Будет более лаконичнее создать файл `index.ts` в корне `src/routes` или `src/pages`
Вот такого содержания
```
import GameRoute  from './GameRoute/gameRoute '
...
export {
GameRoute ,
...
}
```

И из него экспортировать роуты

```suggestion
import {
ErrorPage,
Forum,
...
} from '../routes'
```